public/css/about.css

```
@@ -0,0 +1,180 @@
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 10rem 2rem 4rem;
+    position: relative;
+    z-index: 1;
+    text-align: center;
+}
+
+.about-hero h1 {
+    font-family: 'Orbitron', sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Orbitron" (font-family-name-quotes)

(font-family-name-quotes)